src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => century
+    ? (person.sex === 'm' && Math.ceil(person.died / 100) === century)
+    : person.sex === 'm'
```

Comment:
Duplicated code. Create constants for conditions: `isMan` and `isFromCentury`. Return result of AND operator.